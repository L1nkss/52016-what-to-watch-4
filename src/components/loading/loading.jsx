export const Loading = () => {
  return (
    <div className="testing">
      <div className="lds-ring">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>
  );
};
