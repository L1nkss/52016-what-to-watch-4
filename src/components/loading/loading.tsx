import * as React from "react";

export const Loading = () => {
  return (
    <div className="loading-page">
      <div className="lds-ring">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>
  );
};
