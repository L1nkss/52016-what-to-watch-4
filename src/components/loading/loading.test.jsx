import {Loading} from "./loading";

describe(`Testing Loading component`, () => {
  it(`Component should successfully rendered`, () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
