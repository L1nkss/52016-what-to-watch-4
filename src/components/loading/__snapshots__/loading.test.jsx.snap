// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Loading component Component should successfully rendered 1`] = `
<div
  className="loading-page"
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
